doc/generic/pgf/text-en/pgfmanual-en-library-bbox.tex

```diff
@@ -7,158 +7,164 @@
 %
 % See the file doc/generic/pgf/licenses/LICENSE for more details.
 
-\section{Bounding Boxes for B\'ezier Curves}
-
+\section{Bounding Boxes for Bézier Curves}
```

Comment:
You should ask @tallmarmot whether it's okay to make changes to this file.

Reply:
I had already done that. We agreed that my changes are fine and he will also use this version for further work.